themes/default/content/docs/guides/continuous-delivery/azure-devops.md

```diff
@@ -31,6 +31,21 @@ the steps outlined in the sample YAML file below to the Visual Designer as well.
 - A git repo with your Azure DevOps project set as the remote URL.
     - To learn more about how to [create a git repo in your DevOps project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=vsts&tabs=new-nav).
 
+## Command Summary
+
+| Parameter Name | Required? | Parameter Description |
+|----|---|------|
+| azureSubscription | No | Optionally reference a service connection. If not used, environment variables can be configured with the credentials needed for the applicable Pulumi providers. |
+| command | No | The applicable [Pulumi CLI]({{< ref "/docs/reference/cli" >}}) command. |
+| loginArgs | No | Args to be passed to the [pulumi login command]({{< ref "/docs/reference/cli/pulumi_login" >}}). |
+| args | No | Option flags (e.g. "--yes") that can be passed to the given `pulumi` command. Use space to separate multiple args. For a list of CLI command and supported options, see [Pulumi CLI]({{< ref "/docs/reference/cli" >}}). |
+| cwd | No | The working directory to run the Pulumi commands. Use this if your Pulumi App is in a different directory. |
+| stack | Yes | Name of stack being managed. Can be of the form `ORG/STACK` or `ORG/PROJECT/STACK`. |
+| versionSpec | No | The Pulumi version that should be used. Defaults to the latest version. If you require a specific version then the format is `1.5.0` or if you just need the latest version then `latest` can be used. |
+| createStack | No | Set to `true` if the stack should be created if it does not already exist. Defaults to `false`. |
+| createPrComment | No | Set to `true` to add a comment to your Pull Request (PR). Can only be used in pipelines driven by PRs. Defaults to `false`. See [PR Comments by Task Extension]({{< relref "pr_comments" >}}).
+| useThreadedPrComments | No | Defaults to `true` to always add a comment to the previously-created comments thread. Set to `false` to have each comment added separately.
```

Comment:
```suggestion
| azureSubscription | No | Service connection reference. Alternatively, configure environment variables with Pulumi provider credentials. |
| command | No | The applicable [Pulumi CLI]({{< ref "/docs/reference/cli" >}}) command. |
| loginArgs | No | Args to be passed to the [pulumi login command]({{< ref "/docs/reference/cli/pulumi_login" >}}). |
| args | No | Option flags (e.g. "--yes"), separated by spaces, to pass to the given `pulumi` command.  |
| cwd | No | The working directory to run the Pulumi commands. Use this if your Pulumi App is in a different directory. |
| stack | Yes | Name of stack being managed. Can be of the form `org/stack` or `org/project/stack`. |
| versionSpec | No | The Pulumi version to use. Defaults to the latest version. If you require a specific version then the format is `1.5.0` or if you just need the latest version then `latest` can be used. |
| createStack | No | Boolean indicating if the stack should be created. |
| createPrComment | No | Boolean indicating whether to leave a pull request (PR) comment. Can only be used in pipelines driven by PRs. Defaults to `false`. See [PR Comments by Task Extension]({{< relref "pr_comments" >}}).
| useThreadedPrComments | No | Boolean indicating if PR comments should be threaded. |
```

- im not sure you need all the links to the main CLI docs, maybe one link above the table is enough?
- does `Pulumi App` mean pulumi program?
- may be worth having a defaults column